mangaki/mangaki/templates/fix/fix_suggestion.html

```diff
@@ -0,0 +1,143 @@
+{% extends "base.html" %}
+{% load staticfiles %}
+{% load posters %}
+
+{% block title %}Suggestion #{{ suggestion.pk }} - Mangaki{% endblock %}
+
+{% block body %}
+
+<h1>Suggestion #{{ suggestion.pk }}</h1><br/>
+
+{% if suggestion.is_checked %}
+<div class="alert alert-info">
+    <span class="glyphicon glyphicon-lock" aria-hidden="true"></span> Ce problème a été marqué comme résolu !
+</div>
+{% endif %}
+
+<div class="row">
+
+<div class="col-md-3" align="center">
+    {% include "mangaki/work_rating.html" with category=suggestion.work.category.slug work=suggestion.work simple=True %}
+</div>
+
+<div class="col-md-9">
+    <br/>
+    <div class="row">
+    <div class="col-md-push-1 col-sm-push-1 col-lg-10 col-sm-10">
+        <div class="input-group">
+            <span class="input-group-addon" id="basic-addon2">
+                <span class="glyphicon glyphicon-link"></span> Lien de partage
+            </span>
+            <input type="text" class="form-control" value="{{ request.build_absolute_uri }}">
```

Comment:
I found a handy [CodePen](https://codepen.io/nathanlong/pen/ZpAmjv) so I used it to make it possible to copy the link ;)